<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/recursos/temperatura.png</normaloff>:/recursos/temperatura.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Cent√≠grados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="outCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 240, 107);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 190, 111);</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="value" stdset="0">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inFahr">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="outKelv">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 114, 245);</string>
         </property>
         <property name="value" stdset="0">
          <double>273.149999999999977</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inKelv">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>270</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="pageStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>273</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKelv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKelv</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
